specs/schemes/exact/scheme_exact_algo.md

```diff
@@ -0,0 +1,222 @@
+# Scheme: `exact` on `Algorand`
+
+## Summary
+
+The `exact` scheme on Algorand uses the `Algorand Standard Asset (ASA)`, which are native assets (no contract required) on the Algorand blockchain to authorize a transfer of a specific amount from the payer to the resource server. The approach results in the facilitator having no ability to direct funds anywhere but the address specified by the resource server in `paymentRequirements`.
+
+## Payment Requirements asset definition 
+
+In the `exact` scheme on Algorand, the `paymentRequirements.asset` **MUST** be a string representing an `Algorand Standard Asset (ASA)` ID unsigned integer instead of an `ERC20` contract address. This **MUST** be validated by the resource server to ensure the `asset` field is valid when using the Algorand scheme.
+
+
+## `X-Payment` header payload
+
+The `payload` field of the `X-PAYMENT` header **MUST** contain `paymentGroup` as a field. It represents an atomic group of transactions as an array. This is natively supported by Algorand and does not require contracts.
+
+A group can have transactions of different types, such as, for example, `pay` (payment) and `axfer` (asset transfer). A group in Algorand can contain several different types of transactions. For more info, see the [Algorand documentation on transaction types](https://dev.algorand.co/concepts/transactions/reference/).
+
+As part of the payload there **MUST** also be a `paymentTxID` field which identifies the transaction in the group that will pay the resource server. The group may perform several operations to facilitate the payment, such as swaps or asset transfers, but only one transaction in the group will actually transfer the funds to the resource server.
+
+There can be multiple signers in the group and fees can be pooled together or assigned to a specific signer; meaning it can be delegated to a specific account to pay the fees for the group. A max of 16 transactions can be included in a group.
+
+Example:
+
+```json
+{
+  "paymentTxID": "R4PFEBJGLXV4PDOIBLY7UKBLVLXFCPYLXGBILC22BAVILTKWV3TA", // ID of the transaction in the group that wll pay the resource server
+  "paymentGroup": [
+    {
+      "signature": "wVGxbZwPd2c8zKX4RSE4muFRBOHe/nRq5VT6iMeMksi2agfOCTs+VzOGUPJ+bTT5T21EuYjoBKKDX2+/NZG8Cg==",
+      "txn": {
+        "gen": "mainnet-v1.0",
+        "grp": "abcd1234efgh5678ijkl9012mnop3456qrst7890uvwx1234yzab5678cdef9012",
+        "gh": "SGO1GKSzyE7IEPItTxCByw9x8FmnrCDexi9/cOUJOiI=",
+        "asnd": "PPNHVQJGE6ZMEWPR35UHLUYMCCZV6VNTHLJMZDVCONXKUPV47K4WGQJWGI",
```

Comment:
We need a `snd` field here to be specified. We don't really care about the `asnd` unless it is different from `snd`, but in that case, this transaction would be a "clawback" (which we might be used to "forced" payments and other exotic stuff).